src/main/java/seedu/address/model/person/Task.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.model.person;
+
+import java.util.Objects;
+
+/**
+ * Represents an Item's Task in ELISA.
+ * Completion state of Event is false by default.
+ * Priority of Event is medium by default.
+ * Guarantees: immutable;
+ */
+public class Task {
+
+    private final Priority priority;
+    private final Boolean complete;
+
+    /**
+     * Constructs a {@code Task}.
+     *
+     * @param priority A Priority of the event. Defaults to Priority.MEDIUM if null.
+     * @param complete Denotes whether the task has been completed or not. Defaults to false if null.
+     */
+    public Task(Priority priority, Boolean complete) {
```

Comment:
Perhaps we should consider overloading the constructor again so that null will not be passed into the constructor.